tools/workflow/clean_aws_rsc.go

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package main
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/elbv2"
+	"log"
+	"time"
+	"strings"
+)
+const (
+	containLbString = "aoc-lb"
+	pastDayDelete = 5
+)
+
+func main() {
+
+	log.Printf("Beging terminating EC2 Instances")
+	terminateEc2InstancesError := terminateEc2Instances()
+	if (terminateEc2InstancesError != nil){
+		log.Fatalf("Exit terminating EC2 Instances")
+	}
+
+	log.Printf("Begin destroy Load Balancer resources")
+	destroyLoadBalancerError := destroyLoadBalancerResource()
+
+	if (destroyLoadBalancerError != nil){
+		log.Fatalf("Exit destroy Load Balancer Resources")
+	}
+
+	log.Printf("Finish destroy AWS resources")
+}
+
+func terminateEc2Instances()(error){
+	// set up aws go sdk ec2 client
+	testSession := session.Must(
+		session.NewSessionWithOptions(
```

Comment:
Is all of this necessary? Is the environment variable setup insufficient?
> Creating a Session without additional options will load credentials region, and profile loaded from the environment and shared config automatically.

https://docs.aws.amazon.com/sdk-for-go/api/aws/session/